src/style.css

```diff
@@ -0,0 +1,139 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  --white-color: #fff;
+  --border-color: #f3f3f3;
+  --title-color: #060b35;
+  --subtitle-color: #616070;
+  --color-for-link: #00acdc;
+}
+
+body,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 166px;
+  background-color: var(--white-color);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--title-color);
+}
+
+.card__subtitle {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--subtitle-color);
+  margin-top: 4px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 16px;
+  margin-top: 16px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--title-color);
+}
+
+.card__pricing {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.card__text {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--subtitle-color);
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--title-color);
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--color-for-link);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--white-color);
+  text-decoration: none;
+  border: 1px solid var(--color-for-link);
+  padding: 12px 0;
+  cursor: pointer;
+  margin-top: 16px;
+}
+
+.card__link:hover {
+  color: var(--color-for-link);
+  background-color: var(--white-color);
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
why do we need it if we have the same code in other file?

Reply:
Hello,
I don't understand why you see style.css since I removed it. Instead of style.css there should be index.css and other css files. How I can to resolve this problem?